sections/main-collection-product-grid.liquid

```diff
@@ -351,6 +360,13 @@
                             <label class="field__label" for="Mobile-Filter-{{ filter.label | escape }}-LTE">{{ 'sections.collection_template.to' | t }}</label>
                           </div>
                         </price-range>
+                        <div class="no-js-hidden mobile-facets__footer">
+                          <facet-remove class="mobile-facets__clear-wrapper">
+                            <a href="{{ collection.url }}" class="mobile-facets__clear underlined-link">{{ 'sections.collection_template.clear' | t }}</a>
+                          </facet-remove>
+                          <button type="button" class="no-js-hidden button button--primary" onclick="this.closest('.mobile-facets__wrapper').querySelector('summary').click()">{{ 'sections.collection_template.apply' | t }}</button>
```

Comment:
This is the first time I see a more "elaborated" inline JS code so I'm wondering if it's okay to leave it in Liquid?

Other instances throughout the theme use a more straightforward approach such as `onclick="this.select();"` or `onclick="window.print();"` which I understand because it's not affecting other components.